File: Actividad_03_Pizzera/app/src/main/java/com/example/actividad_03_pizzera/MainActivity.java
package com.example.actividad_03_pizzera;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.util.Log;
import android.widget.Button;
import android.widget.CheckBox;
import android.widget.EditText;
import android.widget.RadioButton;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    private RadioButton grande;
    private RadioButton mediana;
    private RadioButton pequena;

    private CheckBox pepperoni;
    private CheckBox maiz;
    private CheckBox jamon;
    private CheckBox bacon;
    private CheckBox aceitunas;
    private EditText total;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        grande = findViewById(R.id.grande);
        mediana = findViewById(R.id.mediana);
        pequena = findViewById(R.id.pequena);

        pepperoni = findViewById(R.id.pepperoni);
        maiz = findViewById(R.id.maiz);
        jamon = findViewById(R.id.jamon);
        bacon = findViewById(R.id.bacon);
        aceitunas = findViewById(R.id.aceitunas);
        total = findViewById(R.id.totalprecio);

        Button precio = findViewById(R.id.bontonprecio);
        precio.setOnClickListener(view -> calcular());


    }

    @SuppressLint("SetTextI18n")
    private void calcular(){
        int precioBase = 0;

        if(grande.isChecked()){
            precioBase = 15;
        }else if(mediana.isChecked()){
            precioBase = 10;
        }else if(pequena.isChecked()){
            precioBase = 5;
        }

        int precioTotal = precioBase;

        if (pepperoni.isChecked()){
            precioTotal += 2;
        }

        if (maiz.isChecked()){
            precioTotal += 2;
        }

        if (jamon.isChecked()){
            precioTotal += 2;
        }

        if (bacon.isChecked()){
            precioTotal += 2;
        }

        if (aceitunas.isChecked()){
            precioTotal += 2;
        }

        total.setText("€"+precioTotal);
    }
}

File: 00_AppKotlin/app/src/main/java/com/example/a00_appkotlin/MainActivity.kt
package com.example.a00_appkotlin

import android.os.Bundle
import android.util.Log
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

//Para extender de las clases en Kotlin se ponen dos puntos ":"
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var user = Usuario()
        user.nombre = "Felix" //Realmente esta invocando al set
        Log.d("MainActivity", "Nombre: " + user.nombre)//Invocando el get

        val boton1 = findViewById<Button>(R.id.boton1)
        val boton2 : Button = findViewById(R.id.boton2)

        boton1.setOnClickListener({
            Log.d("MainActivity", "Has pulsado el boton 1")
        })

        boton2.setOnClickListener{v ->
            Log.d("MainActivity", "Pulsado boton 2" + v.toString())
            Log.d("MainActivity", "Esto es otra linea")
        }

    }
}